GOVERNANCE.md

```diff
@@ -1,3 +1,246 @@
-## Kuadrant Governance
+# Steering Committee Governance
 
-Governance for the Kuadrant project is still a work in progress. You can participate in discussions via our project [slack channel](https://kubernetes.slack.com/archives/C05J0D0V525).
+The Kuadrant Steering Committee is the governing body of the Kuadrant project,
+providing decision-making and oversight pertaining to the project bylaws,
+sub-organizations, and financial planning. The Steering Committee also defines
```

Comment:
I'd exclude "financial planning" from this sentence and everywhere in the document, in order to avoid misconceptions for the OS community

Reply:
Unless we really want to be the ones managing any kind of budget related to anything Kuadrant (upstream) related.

Reply:
```suggestion
and sub-organizations. The Steering Committee also defines
```

@didierofrivia just removing this seem reasonable?

Reply:
Do note that, as part of the CNCF, the project may have financial resources of its own.  If those are not managed by the Steering Committee, who will manage them?

Reply:
> project may have financial resources of its own

Got an example of how that might come about?

> who will manage them

Good question. Sounds like we need some clause for it here. If not, would it be delegated to some CNCF people?

Reply:
It can't.  The CNCF expects the project to make requests for any money it needs, and to lead any activities which require financing.  The staff won't do it for you.  If nobody in the project does this, they'll just assume you don't need any resources.